core/server/api/app.js

```diff
@@ -68,6 +68,7 @@ function apiRoutes() {
     apiRouter.get('/configuration', api.http(api.configuration.read));
     apiRouter.get('/configuration/:key', authenticatePrivate, api.http(api.configuration.read));
     apiRouter.get('/configuration/timezones', authenticatePrivate, api.http(api.configuration.read));
+    apiRouter.get('/configuration/private', authenticatePrivate, api.http(api.configuration.read));
```

Comment:
This PR works and is fine, however this line, and the line above it, are unnecessary. The `/configuration/:key` route already covers this case.

Would be nice to clean up, but we can merge this and remove them later if preferred